<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Разбор текста</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="main_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Разбор текста с pymorphy2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить файл для разбора</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>931</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="analysis">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Разбор текста</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
